projects/valtimo/exact-plugin/src/lib/exact-plugin.ts

```diff
@@ -0,0 +1,12 @@
+import {PluginConfigurationData} from '@valtimo/plugin';
+
+interface ExactPluginConfig extends PluginConfigurationData {
+  clientId: String;
```

Comment:
just to make sure you're using capitalized 'String' intentionally because you want the type to be an object